support/ruby/collins-client/lib/collins/api/ip_address.rb

```diff
@@ -33,11 +33,12 @@ def ipaddress_update! asset_or_tag, old_address = nil, options = {}
       end
     end
 
-    def ipaddress_delete! asset_or_tag, pool = nil
+    def ipaddress_delete! asset_or_tag, options = {}
```

Comment:
Sure.  I believe the former is the best option there although the latter should be possible thanks to the magic of dynamic types.  Which would you prefer?